### PR TITLE
adjustments to SpatialEntryFilter

### PR DESCRIPTION
**Pull request by banesullivan**
Follow up to #208

I am performing manual testing of the `SpatialEntryFilter` to make sure everything is in working order.

I am hoping to add some basic tests for this as well

### CODE REVIEW

**Comment by mcovalt** [comment]
I've added a single simple test for rasters.

A full fledged test-suite would test each of...
* raster
* FMV
* geometry
 
...for each of the filter options. The result would be tons of tests! I'd like to help contribute more, but I'm curious where we should draw the testing line. Thoughts? Opinions?

File: rgd/geodata/filters.py [hunk 1]
```diff
@@ -120,14 +120,14 @@ def filter_q(self, queryset, name, value):
 
     def filter_predicate(self, queryset, name, value):
         """Filter the spatial entries by the chosen predicate."""
-        if value is not None:
+        if value != '':
```
Comment (banesullivan) [inline rgd/geodata/filters.py:123]:
```suggestion
        if value:
```

File: rgd/geodata/filters.py [hunk 2]
```diff
@@ -120,14 +120,14 @@ def filter_q(self, queryset, name, value):
...
             geom = self.form.cleaned_data['q']
             return queryset.filter(**{f'footprint__{value}': geom})
         return queryset
 
     def filter_relates(self, queryset, name, value):
         """Filter the spatial entries by the chosen DE-9IM."""
-        if value is not None:
+        if value != '':
```
Comment (banesullivan) [inline rgd/geodata/filters.py:130]:
```suggestion
        if value:
```

**Comment by banesullivan** [comment]
I think that these basic tests are all we should worry about for now. Thanks, @mcovalt! In time, we can implement full coverage for the `filters.py` module but as long as we can verify the geometry/predicate work then I think that is sufficient.

I will add one additional test using GeoJSON and then merge for us to add more comprehensive tests later.

**Pull request merged**